### DISCUSSION TITLE
Opening Document, but Viewer still on old document?

### DISCUSSION THREAD

**Discussion by 0lm** [vkbo/novelWriter#1562] [Help] [answered]
Hi. When I double click on a document in the tree, it opens to be edited, but on the right side, the viewed dicument (bascially the results of the edits) are still the very first document. To be able to switch it, I always have to right-click on the document and click "view document" to make it appear on the right-side viewer, too.

Is there a setting that automaitcally does both, opens the document in the editor and in the viewer, by just double clicking a document? I couldnt find it.

**Comment by vkbo** [comment, marked as answer]
The document viewer is to view your notes, it's not for previewing your edits. Although you can use it for that if you wish. If you press `Ctrl+R` while editing, the viewer will load the document your writing and update the content.

See the documentation: https://novelwriter.io/docs/usage_writing.html#viewing-a-document
